src/components/Footer/Final_pend.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components';
+
+const CradleContainer = styled.div`
+  position: absolute;
+  width: 200px;
+  height: 40px;
+  padding-top: 100px;
+  color: white;
+  position: absolute;
+`;
+
+const CradleBefore = styled.div`
+  content: '';
+  display: block;
+  position: absolute;
+  width: 200px;
+  height: 6px;
+  top: 0;
+  left: 0;
+  border-radius: 3px;
+  background: #bdc3c7;
+`;
+
+const Ball = styled.div`
+  position: relative;
+  float: left;
+  width: 40px;
+  height: 40px;
+  background: white;
+  border-radius: 50%;
+  transform-origin: 50% -100px;
+  &:before {
+    content: '';
+    display: block;
+    position: absolute;
+    height: 200px;
+    width: 1px;
+    top: -100px;
+    left: 19px;
+    background: #bdc3c7;
+  }
+`;
```

Comment:
Move the styles to `styles.js`

Reply:
moved. [commited](https://github.com/dscnitrourkela/project-oregano/pull/241/commits/b5fe63a6079822c58f7e5b74b4f79047342b44fa)